aep/general/0205/aep.md.j2

```diff
@@ -1,5 +1,47 @@
 # Beta-blocking changes
 
-**Note:** This AEP has not yet been adopted. See
-[this GitHub issue](https://github.com/aep-dev/aep.dev/issues/50) for more
-information.
+APIs often release an Alpha version of their API in order to get early feedback
+from customers. This API is provisional and can change many times before the
+important feedback is incorporated and the API is made stable for Beta.
+
+Since the purpose of Alpha is to gather feedback, the API does not need to be
+perfect yet, and it's not strictly necessary for API authors to address every
+usability concern or address every point in the API standards. Often, API
+authors and API reviewers will not agree on the best design, and the best way
+to find out is by having users try out the API.
+
+However, once the feedback has been collected and the API is going to be
+promoted to Beta, usability concerns and style issues do need to be addressed.
+In order to ensure that these issues are not forgotten, they should be
+explicitly documented in the API.
+
+## Guidance
+
+If an API has usability concerns or violates API standards, and the present
+design should receive additional scrutiny before being carried through to the
+Beta version, there **must** be an internal comment linking to this document
+using its descriptive link ([aep.dev/beta-blocker]()) to ensure that the design
+is corrected before the API is released to Beta.
+
+The comment **must** also indicate what kind of change should be made for Beta.
+For example:
+
+{% tab proto %}
+
+```proto
+message InputConfig {
+  // Parameters for input.
+  // (-- aep.dev/beta-blocker: Convert well-known parameters into explicit
+  //     fields before the Beta launch. --)
+  map<string, string> parameters = 1;
+}
+```
+
+{% tab oas %}
+
+**Note:** OAS example not yet written.
```

Comment:
can you file an issue on these so we remember where they are? folks may be interested in picking these up as well, post barn-raising.

Reply:
Yes - but if you're okay with it, I'd like to do it as a batch process (to make sure I catch everything) decoupled from any specific PR.

Reply:
sure, but I think we have to be reasonable about the "batches" - it's unlikely that someone is going to do every single missing OAS example in one PR.

Can you make it granular to the AEP? IF you file an issue go ahead and resolve.